packages/SwingSet/src/kernel/state/vatKeeper.js

```diff
@@ -25,10 +25,10 @@ const FIRST_TRANSCRIPT_ID = 0;
  * TODO: consider making this part of makeVatKeeper
  */
 export function initializeVatState(storage, vatID) {
-  storage.set(`${vatID}.o.nextID`, `${FIRST_OBJECT_ID}`);
-  storage.set(`${vatID}.p.nextID`, `${FIRST_PROMISE_ID}`);
-  storage.set(`${vatID}.d.nextID`, `${FIRST_DEVICE_ID}`);
-  storage.set(`${vatID}.t.nextID`, `${FIRST_TRANSCRIPT_ID}`);
+  storage.set(`${vatID}.o.nextID`, toStr(FIRST_OBJECT_ID));
+  storage.set(`${vatID}.p.nextID`, toStr(FIRST_PROMISE_ID));
+  storage.set(`${vatID}.d.nextID`, toStr(FIRST_DEVICE_ID));
+  storage.set(`${vatID}.t.nextID`, toStr(FIRST_TRANSCRIPT_ID));
```

Comment:
You created a counter abstraction in `kernelKeeper.js`.  Why not use that here?

Reply:
I removed the counter abstraction to get this PR merged more quickly, as @warner seemed to prefer using the string keys directly